the-devastating-chickens/BangazonAPI
# CRUD on Products, Testing Green

# Description

The products controller, class, and testing are completed.

Fixes Issue #4 

## Type of change



# Testing Instructions

1. git fetch --all
2. git checkout mm-Products
3. Run all tests on TestProducts and confirm green

# Checklist:
